app/models/manageiq/providers/openstack/helper_methods.rb

```diff
@@ -24,8 +24,8 @@ def openstack_proxy
     end
 
     def parse_error_message_from_fog_response(exception)
-      exception_string = exception.to_s
-      matched_message = exception_string.match(/message\\\": \\\"(.*)\\\", /)
+      exception_string = exception.respond_to?(:response) ? exception.response&.body : exception.to_s
+      matched_message = exception_string.match(/"message": "(.*)"/)
```

Comment:
This match might be too greedy _or_ possibly not greed enough depending on the content.  1st question - can the message have embedded `"` characters?

Reply:
Yeah so this wasn't matching any of the exceptions that I was getting back:
```
exception.to_s.match(/message\\\": \\\"(.*)\\\", /) # The original
nil
exception_string.match(/message\\\": \\\"(.*)\\\", /) # Pulling the body out of the exception
nil
```

Here is an example of `exception.response.body`: `"{\"badRequest\": {\"code\": 400, \"message\": \"Invalid volume: Volume to be backed up must be available or in-use, but the current status is \\\"backing-up\\\".\"}}"`